src/util/externalCommand.ts

```diff
@@ -0,0 +1,121 @@
+import { exec } from 'child_process';
```

Comment:
node APIs should be isolated so they can be replaced at a later date (see #4981 and let me know if you've got any questions about that)